setup.py

```diff
@@ -18,5 +18,8 @@
           "Programming Language :: Python :: 2",
           "Programming Language :: Python :: 3",
           "Topic :: Scientific/Engineering",
+      ],
+      data_files=[
```

Comment:
The problem is that the script is used by the library itself (it is not an optional or external script). If you don't install it, the blender importer will not work at all. And if you try to call it from the repository, how does the installed library can find the script ?